model/src/main/java/org/cbioportal/model/ResourceDefinition.java

```diff
@@ -0,0 +1,75 @@
+package org.cbioportal.model;
+
+import javax.validation.constraints.NotNull;
+
+public class ResourceDefinition {
+
+    @NotNull
+    private String resourceId;
+    @NotNull
+    private String displayName;
```

Comment:
All looks good to me! Just the `displayName` maybe should be nullable?

Reply:
Added a comment in database pr https://github.com/cBioPortal/cbioportal/pull/7259/files#r388315615. Let's discuss it there

Reply:
done